<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="leftList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <item>
        <property name="text">
         <string>Bubble charts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tree views</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Import data</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="3">
           <widget class="QSpinBox" name="spDefaultVisibleChartTaxes">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Default maximum bubble size</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Default number of visible items in chart</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="sbDefaultMaxBubbleSize">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="sbMaxChartTax">
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Maximum number of items in chart</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Node circle size</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Maximum node radius</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Plus sign size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="sbNodeCircleSize">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="sbMaxNodeRadius">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="sbPlusSignSize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>70</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Edge style:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QRadioButton" name="rbLines">
            <property name="text">
             <string>&amp;Lines</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QRadioButton" name="rbCurves">
            <property name="text">
             <string>&amp;Curves</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="3">
           <widget class="QPushButton" name="bGiToTaxMapPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="leGiToTaxMapPath"/>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>&amp;Gi to taxonomy map</string>
            </property>
            <property name="buddy">
             <cstring>bGiToTaxMapPath</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="sbMinBitscore">
            <property name="value">
             <double>70.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Min &amp;Bitscore</string>
            </property>
            <property name="buddy">
             <cstring>sbMinBitscore</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>&amp;Top percent</string>
            </property>
            <property name="buddy">
             <cstring>sbTopPercent</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="sbTopPercent"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leftList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbNodeCircleSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbPlusSignSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbMaxNodeRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbMaxChartTax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spDefaultVisibleChartTaxes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbDefaultMaxBubbleSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbMinBitscore</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leGiToTaxMapPath</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbTopPercent</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>onConfigChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>onListSelectionChanged(QModelIndex)</slot>
  <slot>onConfigChanged()</slot>
 </slots>
</ui>
